### PR TITLE
Add osu!taiko pp committee members.

### PR DESCRIPTION
**Pull request by mangomizer**
I'm pretty sure "-" is before A-Z so Lawtron is first.
There may be more members added in the near future, but these are the ones currently added.

### CODE REVIEW

**Comment by mangomizer** [comment]
Out of curiosity, would it make sense to link their respective GitHubs as well, or is osu profile just sufficient?

**Comment by Walavouchey** [comment]
the github team listing only has Mango and Lawtron added, so i'll wait for that to get sorted

> Out of curiosity, would it make sense to link their respective GitHubs as well, or is osu profile just sufficient?

sure

File: wiki/People/Performance_Points_Committee/en.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ The **Performance Points Committee** is a panel of well-versed players and devel
 
 The osu! panel was first announced to the public through the [November 2021 *Performance Points & Star Rating Updates* news post](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to a [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) held earlier that year.
 
-osu!mania's panel was formed 22 April 2022 in anticipation of a surge in active development effort, starting with [a change in difficulty calculation for hold note releases in osu!mania](https://github.com/ppy/osu/pull/17913).
+The osu!mania panel was formed 22 April 2022 in anticipation of a surge in active development effort, starting with [a change in difficulty calculation for hold note releases in osu!mania](https://github.com/ppy/osu/pull/17913).
+
+The osu!taiko panel was formed on 1 May 2022 for reasons similar to the above, with the first batch of changes expected to roll out shortly.
```
Comment (Lawtrohux) [inline wiki/People/Performance_Points_Committee/en.md:14]: The reason this is in a comment is I have two recommendations:

The osu!taiko panel was formed on 1 May 2022 in response to a [community survey](https://osu.ppy.sh/home/news/2022-01-14-taiko-mania-pp-sr-survey) held earlier that year, starting with a [document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit?usp=sharing) outlining the information of the changes to difficulty calculation.

Or, if a outside document cannot be linked within the wiki then:

The osu!taiko panel was formed on 1 May 2022 in response to a [community survey](https://osu.ppy.sh/home/news/2022-01-14-taiko-mania-pp-sr-survey) held earlier that year.

Comment (Walavouchey) [reply wiki/People/Performance_Points_Committee/en.md:14]: second option sounds fine

**Comment by Walavouchey** [comment]
> the github team listing only has Mango and Lawtron added, so i'll wait for that to get sorted

@smoogipoo do you happen to know the status on this?

**Comment by smoogipoo** [comment]
@Walavouchey One of the members didn't have a GitHub account at the time, or so I was told. They've since been added and I believe the member lists now match.

**Comment by Lawtrohux** [comment]
@mangomizer do we still want to add our github profiles?

**Comment by Walavouchey** [comment]
i've linked the document since it looks fleshed out enough and seems to be directed towards a general audience anyway based on how it's written

**Review by Walavouchey — DISMISSED** [review]
lgtm @mangomizer @Lawtrohux if there's anything else do tell

File: wiki/People/Performance_Points_Committee/en.md [hunk 2]
```diff
@@ -9,30 +9,37 @@ The **Performance Points Committee** is a panel of well-versed players and devel
...
-- ![][flag_GB] [Apo11o](https://osu.ppy.sh/users/9558549)
-- ![][flag_HU] [emu1337](https://osu.ppy.sh/users/2185987)
-- ![][flag_AU] [MBmasher](https://osu.ppy.sh/users/4498616)
-- ![][flag_RU] [StanR](https://osu.ppy.sh/users/7217455)
+| osu! profile | GitHub profile |
+| :-- | :-- |
+| ![][flag_GB] [Apo11o](https://osu.ppy.sh/users/9558549) | [apollo-dw](https://github.com/apollo-dw) |
+| ![][flag_HU] [emu1337](https://osu.ppy.sh/users/2185987) | [emu1337](https://github.com/emu1337) |
+| ![][flag_AU] [MBmasher](https://osu.ppy.sh/users/4498616) | [MBmasher](https://github.com/MBmasher) |
+| ![][flag_RU] [StanR](https://osu.ppy.sh/users/7217455) | [standriders](https://github.com/stanriders) |
```
Comment (apollo-dw) [inline wiki/People/Performance_Points_Committee/en.md:25]:
```suggestion
| ![][flag_RU] [StanR](https://osu.ppy.sh/users/7217455) | [stanriders](https://github.com/stanriders) |
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**